go/libkb/vdebug.go

```diff
@@ -32,26 +35,57 @@ const (
 	VLog1
 	VLog2
 	VLog3
+
+	VLogNoneString       = "mobile"
```

Comment:
Is the plan to have "mobile" equivalent to None forever?

Reply:
No clue -- I'm just rearranging what was already there.  I don't think it matters right now.